rca/static_src/sass/base/_typography.scss

```diff
@@ -34,10 +34,6 @@ blockquote {
     text-transform: inherit;
 }
 
-h1 {
-    @include font-size(xxl);
-}
-
```

Comment:
we don't need this?

Reply:
We're going to move these classes into a rich-text component, so that we're not having to override sizes on every h tag with heading heading--modifier.